backends/bmv2/common/action.cpp

```diff
@@ -181,7 +181,7 @@ void ActionConverter::convertActionBody(const IR::Vector<IR::StatOrDecl> *body,
                 continue;
             }
         }
-        ::error(ErrorType::ERR_UNSUPPORTED, "%1% not yet supported on this target", s);
+        ::P4::error(ErrorType::ERR_UNSUPPORTED, "%1% not yet supported on this target", s);
```

Comment:
We can leave it for this PR but maybe we should change the way this function is invoked. Either qualify it properly or remove the `::P4::` prefix.

Reply:
Agree this is worth addressing in the future. In general I feel some more careful per-subdirectory namespace usage cleanup work is needed as follow-up work.

Reply:
Yeah, `::P4::error` looks quite long to me :)